changelog.d/.towncrier_template.md.jinja

```diff
@@ -0,0 +1,70 @@
+{%- set is_draft_preview = versiondata["version"] == "DRAFT_VERSION" -%}
+
+{%- if is_draft_preview -%}
+## Unreleased Changes
+{% else -%}
+## {{ versiondata["version"] }}
+{% endif %}
+
+*{%- if is_draft_preview -%}Generated for preview on {% endif -%}{{ versiondata["date"] }}*
```

Comment:
I just now realized that Hynek even added a warning admonition for this: https://github.com/python-attrs/attrs/blob/199cd548e11197147b9e7438e13c11c621b7b173/changelog.d/towncrier_template.md.jinja#L4-L6. And I remembered he also wrapped the version titles as links to GitHub (I saw this during the initial integration in there since he didn't know it was possible to use my extension with Markdown initially so I ended up watching the progress but forgot that by now). Pretty neat! Perhaps, we need to look into doing the same later.

Reply:
It might be possible to do more by following his example, since I see that he's unconditionally including the towncrier content ( https://github.com/python-attrs/attrs/blob/199cd548e11197147b9e7438e13c11c621b7b173/docs/changelog.md?plain=1#L5 ).

Let's circle back on this after the release. I think his setup may have multiple things we can try out.